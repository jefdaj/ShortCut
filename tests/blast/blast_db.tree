.
|-- cache
|   |-- each
|   |   `-- c56e6b4ee8
|   |       |-- 124e9063de.str.list -> ../../../exprs/extract_queries/124e9063de.str.list
|   |       |-- 124e9063de.str.list.args -> ../../../cache/lines/...
|   |       |-- 1c78118a6f.str.list -> ../../../exprs/extract_queries/1c78118a6f.str.list
|   |       |-- 1c78118a6f.str.list.args -> ../../../cache/lines/...
|   |       |-- 217d76f66f.str.list -> ../../../exprs/extract_queries/217d76f66f.str.list
|   |       |-- 217d76f66f.str.list.args -> ../../../cache/lines/...
|   |       |-- 4dcfad3c69.str.list -> ../../../exprs/extract_queries/4dcfad3c69.str.list
|   |       |-- 4dcfad3c69.str.list.args -> ../../../cache/lines/...
|   |       |-- 934a55faaa.str.list -> ../../../exprs/extract_queries/934a55faaa.str.list
|   |       |-- 934a55faaa.str.list.args -> ../../../cache/lines/...
|   |       |-- e9b730c3ee.str.list -> ../../../exprs/extract_queries/e9b730c3ee.str.list
|   |       `-- e9b730c3ee.str.list.args -> ../../../cache/lines/...
|   |-- load
|   |   |-- 0238b7dcea.fna
|   |   |-- 0238b7dcea.fna.ids
|   |   |-- 32841598b2.faa
|   |   |-- 32841598b2.faa.ids
|   |   |-- 6ea57f1208.fna
|   |   |-- 6ea57f1208.fna.ids
|   |   |-- 8a695c317a.faa
|   |   |-- 8a695c317a.faa.ids
|   |   |-- 8aa4a9d2fc.faa
|   |   |-- 8aa4a9d2fc.faa.ids
|   |   |-- a810086108.faa
|   |   |-- a810086108.faa.ids
|   |   |-- ff08c7f09e.fna
|   |   `-- ff08c7f09e.fna.ids
|   `-- makeblastdb
|       |-- a1512fc008
|       |   |-- a1512fc008.pdb.phr
|       |   |-- a1512fc008.pdb.pin
|       |   `-- a1512fc008.pdb.psq
|       |-- d24c42156b
|       |   |-- d24c42156b.ndb.nhr
|       |   |-- d24c42156b.ndb.nin
|       |   `-- d24c42156b.ndb.nsq
|       `-- e00fb07ddb
|           |-- e00fb07ddb.pdb.phr
|           |-- e00fb07ddb.pdb.pin
|           `-- e00fb07ddb.pdb.psq
|-- exprs
|   |-- any
|   |   `-- eb0be40455.str.list -> ../../cache/lines/...
|   |-- blastn_db
|   |   |-- 8158675169_8df2fbb1a8_d326247d21.bht
|   |   `-- 8158675169_bc4ea1fc06_d326247d21.bht
|   |-- blastp_db
|   |   |-- 8158675169_1af4d085e7_67b1500345.bht
|   |   `-- 8158675169_6e8bd8b7c6_46b5852d89.bht
|   |-- blastx_db
|   |   |-- 8158675169_8df2fbb1a8_46b5852d89.bht
|   |   `-- 8158675169_bc4ea1fc06_67b1500345.bht
|   |-- extract_queries
|   |   |-- 124e9063de.str.list -> ../../cache/lines/...
|   |   |-- 1c78118a6f.str.list -> ../../cache/lines/...
|   |   |-- 217d76f66f.str.list -> ../../cache/lines/...
|   |   |-- 4dcfad3c69.str.list -> ../../cache/lines/...
|   |   |-- 934a55faaa.str.list -> ../../cache/lines/...
|   |   `-- e9b730c3ee.str.list -> ../../cache/lines/...
|   |-- extract_queries_each
|   |   `-- 5d3bf39227.str.list.list -> ../../cache/lines/...
|   |-- list
|   |   |-- 2dd3f7f06e.faa.list -> ../../cache/lines/...
|   |   |-- 6dc0612e56.bht.list -> ../../cache/lines/...
|   |   |-- c5595751e2.faa.list -> ../../cache/lines/...
|   |   `-- e8624f515c.fna.list -> ../../cache/lines/...
|   |-- load_faa
|   |   |-- 2d88028e40.faa -> ../../cache/load/32841598b2.faa
|   |   |-- 387f499ddc.faa -> ../../cache/load/a810086108.faa
|   |   |-- bb54df8bd7.faa -> ../../cache/load/8a695c317a.faa
|   |   `-- dee24d6c3c.faa -> ../../cache/load/8aa4a9d2fc.faa
|   |-- load_fna
|   |   |-- 82b637da24.fna -> ../../cache/load/0238b7dcea.fna
|   |   |-- 9b7934c579.fna -> ../../cache/load/6ea57f1208.fna
|   |   `-- eb12539f1c.fna -> ../../cache/load/ff08c7f09e.fna
|   |-- makeblastdb_nucl
|   |   `-- 17e9ac9897.ndb -> ../../cache/lines/...
|   |-- makeblastdb_prot
|   |   |-- 27a934c58e.pdb -> ../../cache/lines/...
|   |   `-- ced061f0b2.pdb -> ../../cache/lines/...
|   |-- num
|   |   `-- a53b190835.num -> ../../cache/lines/...
|   `-- str
|       |-- 1d8a61c770.str -> ../../cache/lines/...
|       |-- 4cdbcfa070.str -> ../../cache/lines/...
|       |-- 50dbefb6d6.str -> ../../cache/lines/...
|       |-- 7da80bde02.str -> ../../cache/lines/...
|       |-- 8fa8ede325.str -> ../../cache/lines/...
|       |-- db90f32127.str -> ../../cache/lines/...
|       `-- ec3b181d8a.str -> ../../cache/lines/...
`-- vars
    |-- hits6803n.bht -> ../exprs/blastn_db/8158675169_8df2fbb1a8_d326247d21.bht
    |-- hits6803p.bht -> ../exprs/blastp_db/8158675169_6e8bd8b7c6_46b5852d89.bht
    |-- hits6803x.bht -> ../exprs/blastx_db/8158675169_8df2fbb1a8_46b5852d89.bht
    |-- hits7942n.bht -> ../exprs/blastn_db/8158675169_bc4ea1fc06_d326247d21.bht
    |-- hits7942p.bht -> ../exprs/blastp_db/8158675169_1af4d085e7_67b1500345.bht
    |-- hits7942x.bht -> ../exprs/blastx_db/8158675169_bc4ea1fc06_67b1500345.bht
    |-- large6803faa.faa -> ../exprs/load_faa/dee24d6c3c.faa
    |-- large7942faa.faa -> ../exprs/load_faa/387f499ddc.faa
    |-- large7942fna.fna -> ../exprs/load_fna/eb12539f1c.fna
    |-- result -> ../exprs/any/eb0be40455.str.list
    |-- small6803faa.faa -> ../exprs/load_faa/bb54df8bd7.faa
    |-- small6803fna.fna -> ../exprs/load_fna/9b7934c579.fna
    |-- small7942faa.faa -> ../exprs/load_faa/2d88028e40.faa
    `-- small7942fna.fna -> ../exprs/load_fna/82b637da24.fna

23 directories, 85 files
