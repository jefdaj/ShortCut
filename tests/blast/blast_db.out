["Synpcc7942_0003",
 "Synpcc7942_0006",
 "AOY38_00015",
 "Synpcc7942_0010",
 "Synpcc7942_0010",
 "AOY38_00040",
 "Synpcc7942_0005",
 "Synpcc7942_0004",
 "Synpcc7942_0009",
 "SYNGTS_0010",
 "SYNGTS_0005",
 "SYNGTS_0007",
 "Synpcc7942_0001",
 "SYNGTS_0004",
 "AOY38_00030",
 "AOY38_00035",
 "Synpcc7942_0003",
 "SYNGTS_0006",
 "AOY38_00050",
 "AOY38_00010",
 "Synpcc7942_0007",
 "Synpcc7942_0008",
 "SYNGTS_0001",
 "SYNGTS_0008",
 "SYNGTS_0003",
 "AOY38_00020",
 "AOY38_00045",
 "SYNGTS_0009",
 "Synpcc7942_0002",
 "Synpcc7942_0001",
 "Synpcc7942_0005",
 "SYNGTS_0002",
 "AOY38_00025",
 "AOY38_00005"]
